sys/sys/fcntl.h

```diff
@@ -229,6 +229,7 @@ typedef	__pid_t		pid_t;
 #endif	/* __POSIX_VISIBLE >= 200809 */
 #if __BSD_VISIBLE
 /* #define AT_UNUSED1		0x1000 *//* Was AT_BENEATH */
+#define	AT_UTIMENSAT_FULL	0x1000
```

Comment:
I don't like the name FULL as not specific enough.  What if one day one more is added?  I'd prefer BTIME here.